tools/ts_library_builder/build_library.ts

```diff
@@ -419,7 +434,6 @@ export function main({
     compilerOptions: {
       baseUrl: buildPath,
       moduleResolution: ModuleResolutionKind.NodeJs,
-      noLib: true,
```

Comment:
ditto - is this required for arm builds?

Reply:
ditto